app/controllers/media/objects_controller.rb

```diff
@@ -15,6 +15,22 @@ def show
       filename: media_filename
   end
 
+  def create
+    media = media_class(params[:type]).new(item: params[:upload])
+    media.answer = Answer.find(params[:answer_id]) if params[:answer_id]
```

Comment:
Perhaps a comment here explaining why it's ok to leave answer_id blank. It's not super clear.